storage/src/mmr/benches/prove_single_element.rs

```diff
@@ -1,5 +1,5 @@
 use commonware_cryptography::{Hasher, Sha256};
-use commonware_storage::mmr::mem::Mmr;
+use commonware_storage::mmr::{mem::Mmr, verification::VerifiableMmr};
```

Comment:
nit: I wonder if there is a clean way to put the in-memory MMR at `mmr::Memory` (`mmr::mem::Mmr` seems suboptimal)